src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  padding: $el-padding;
+  max-width: $el-size*7 + 7*$el-gap;
```

there are only 6 gaps between 7 days, also add some spaces
```suggestion
  max-width: $el-size * 7 + 6 * $el-gap;
```